src/pages/Invoice/CreateInvoice.js

```diff
@@ -120,17 +122,16 @@ const CreateInvoice = () => {
                     <div className="page-heading-wrapper mb-5 p-5">
                         <span className="title">
                             {" "}
-                            <FormattedMessage id="title.invoice"></FormattedMessage>{" "}
+                            <FormattedMessage id="title.invoice" />
                         </span>
-                        <button className="btn" type="submit">
-                            <i className="fa fa-save"></i> &nbsp;{" "}
-                            <FormattedMessage id="invoice.save.button"></FormattedMessage>
-                        </button>
+                        <Button className="btn" icon="save" type="submit">
+                            <FormattedMessage id="invoice.saveButton" />
+                        </Button>
                     </div>
                     <div className="d-flex py-5 flex-grow align-items-start">
                         <div className="card-bordered p-3 mx-5">
                             <h4 className="bill-to text-muted m-0 mb-3">
-                                <FormattedMessage id="invoice.bill.to"></FormattedMessage>{" "}
+                                <FormattedMessage id="invoice.billTo" />
```

Comment:
Every where you used . notiation in naming id's , here why we making a different pattern of camel case?